src/www/components/landingpage/LandingPage.scss

```diff
@@ -0,0 +1,612 @@
+.container {
+  background-image: linear-gradient(  rgb(129, 129, 129) ,white);
+  height: 325vh; //parallax(110vh) + 2 pages(200vh) + 15vh more empty space
+  max-width: 100vw;
+  align-items: center;
+  justify-content: center;
+  font-family: 'Times New Roman', Times, serif;
+
+  .thirdpage{
+    
+    .phrase{
+      margin-top: 20vh;
```

Comment:
I feel like this is too much vertical gap (same applies to all "header" elements).